### PR TITLE
ref: Eliminate implicit jQuery usage

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Ensurers jQuery is imported as '$' when it is used. Do not support implicitly using jQuery within modles

Removes some usage of `jQuery.parmas`.

It's important to note that jQuery is *still* made available on the `window` as it is exported by the sentry app/index entrypoint.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/utils.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import {isString} from 'lodash';
 import * as Sentry from '@sentry/browser';
+import queryString from 'query-string';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/interfaces/utils.jsx]: I think we generally import this as `qs?` but *shrug* that's default imports for ya.

**Review by billyvg — APPROVED** [review]
We should remove jquery as a global in our eslint conf https://github.com/getsentry/eslint-config-sentry/blob/81f6a8d8f3a8c94a2d56139a2eef501c9a86f82b/packages/eslint-config-sentry-app/index.js

**Comment by EvanPurkhiser** [comment]
I have a fix for the build (adding back the provider plugin :( ) but I want to merge https://github.com/getsentry/sentry/pull/11475 first

**Comment by EvanPurkhiser** [comment]
I'm not sure why precy is being so unhappy with this PR.

**Comment by billyvg** [comment]
Is there anything in our acceptance tests that rely on global jquery?

**Comment by EvanPurkhiser** [comment]
Going to try creating a new PR :/

**Pull request closed without merging**